pennylane/beta/tapes/qnode.py

```diff
@@ -418,6 +422,18 @@ def to_torch(self, dtype=None):
 
             self.dtype = dtype or self.dtype or TorchInterface.dtype
 
+            if self.dtype is np.complex128:
+                torch_version = list(map(int, torch.__version__.split(".")[:2]))
+                min_version_for_complex = torch_version[0] >= 1 and torch_version[1] >= 6

```

Comment:
Since `semantic-version` is a dependency of PennyLane, could we use that to determine this? (I actually tried that in the past elsewhere and couldn't get it to work